lib/greed/turn.rb

```diff
@@ -0,0 +1,63 @@
+absolute_path = File.expand_path(File.dirname(__FILE__))
+require absolute_path + '/dice'
+require absolute_path + '/score'
+require 'pry'
+
+module Greed
+  class Turn
+
+    def initialize
+      @dice = Dice.new
+      @score = Score.new
+    end
+
+    # Run a full single turn for a player until they choose to stop, or score 0
+    def take_turn
+      score = 0
+      # Make the initial roll
+      roll(5)
+      result = @score.calc_score(@dice.values)
+      puts "Your initial roll is: #{@dice.values}."
```

Comment:
Why does `Turn` have responsibility for printing stuff to the screen. You should take a look at https://en.wikipedia.org/wiki/Single_responsibility_principle and think about what responsibilities your classes each have.

What if I wanted to make this use a web interface, rather than a command line interface? What portion of your game logic could I re-use?